viewing_party/party.py

```diff
@@ -1,23 +1,227 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from audioop import avg
+from sys import argv

```

Comment:
I don't see these libraries were used?

Reply:
Thank you Jerica for what I did well on and on area's to improve! I'm not sure what happened with the libraries? I saw they were greyed out and definitely had it in my mind to ask but soon as the tests started passing I think it completely slipped my mind! I will go back and check for this project and keep all this in mind for the future. Thank you!